test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -576,11 +576,9 @@ def forward(self, input):
                 return input.view(1, -1)
 
         x = torch.ones(2)
-        another_x = torch.empty((0,))
```

Comment:
This is a dynamic shape test. It is guarding the scenario in which a model exported with 1d input can work with a 0d later on.

If this PR forces the behavior to be static, then we need to delete the whole test, not only this line. Otherwise, the test will not test for "dynamicness" anyway

Reply:
If you could file an issue letting us know that this test is gone, we can look into it later and see how we can replace it with another one